ui-spacetimechart/src/components/workSchedules/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default as WorkSchedules } from './WorkSchedules';
+export { default as Region } from './Region';
```

Comment:
Is `Region` something we really want to expose to library users and maintain as public API?